playbooks/ansible-cloud/vcenter-appliance/files/bootstrap.yaml

```diff
@@ -24,29 +30,17 @@
 
     - name: Generate random password
       set_fact:
-        __password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters') }}"
+        __password: "{{ lookup('password', '/dev/null length=15 chars=ascii_letters,digits,hexdigits') }}"
       no_log: true
 
-    - name: Change zuul user password
+    - name: Change zuul user password (vcenter)
       become: true
       user:
         name: zuul
         password: "{{ __password | password_hash('sha512') }}"
+      when: inventory_hostname == "vcenter"
 
-- hosts: vcenter
-  gather_facts: false
-  tasks:
-    # NOTE(Gonéri): ok, some explanations here. We can set the hostname of the vcenter node
-    # during the installation, and we can hardly change it after. So we configure the VM
-    # to use dnsmasq internally and use /etc/hosts as a source of trust.
-    # Also, the hostname has to be in the format $hostname.$domain. In addition, some services
-    # use the reverse DNS resolution to know the public IP of the node. So we cannot just go
-    # with "vcenter.test==127.0.0.1", this won't work.
-    - name: Write the /etc/hosts file
-      copy:
-        content: |
-          127.0.0.1  localhost
-          ::1  localhost ipv6-localhost ipv6-loopback
-          {{ hostvars['vcenter']['nodepool']['interface_ip']}} vcenter.test vcenter
-        dest: /etc/hosts
-      become: true
+    - name: Change zuul user password (ESXi)
```

Comment:
understand what we are doing, but we are kinda blurring the line here of this being the 'vcenter bootstrap playbook', can we move this into esxi specific playbooks?

Reply:
I drop this playbook, it overlaps with `vmware-ci-set-passwords` anyway.

Reply:
Regarding the `Set SSH interface IP for appliance` comment, isn't something that nodepool already check?